djstripe/models/billing.py

```diff
@@ -1393,6 +1393,17 @@ class Subscription(StripeModel):
         "invoice items will be billed according to the schedule provided at "
         "pending_invoice_item_interval.",
     )
+    pause_collection_resumes_at = StripeDateTimeField(
```

Comment:
We're not using decomposed fields like this. You can just add the missing field as `StripeJSONField`.